lib/swaps/Swaps.ts

```diff
@@ -561,26 +561,25 @@ class Swaps extends EventEmitter {
     }
 
     if (height) {
-      this.logger.debug(`got block height of ${height} for ${takerCurrency}`);
+      this.logger.debug(`got ${takerCurrency} block height of ${height}`);
 
       const routeAbsoluteTimeLock = deal.makerToTakerRoutes[0].getTotalTimeLock();
-      this.logger.debug(`choosing a route with total time lock of ${routeAbsoluteTimeLock}`);
       const routeTimeLock = routeAbsoluteTimeLock - height;
-      this.logger.debug(`route time lock: ${routeTimeLock}`);
+      this.logger.debug(`choosing a maker to taker route with total time lock of ${routeTimeLock} (${takerCurrency})`);
 
       const makerClientLockBuffer = this.swapClientManager.get(makerCurrency)!.lockBuffer;
-      this.logger.debug(`maker client lock buffer: ${makerClientLockBuffer}`);
+      this.logger.debug(`maker client lock buffer: ${makerClientLockBuffer} (${makerCurrency})`);
```

Comment:
Should the lockBuffer be expressed in blocks of a currency? is it different between currencies?

Reply:
Yes, it's a number of blocks and it's based on the block time for that chain.

Reply:
I know we currently express it in blocks. 
However, I think that the intention was to add 1 day buffer. If this is the case we should have it as a const/config and calcualte the number of blocks that we should add to the first leg.

Reply:
It is expressed in hours in the **config** since https://github.com/ExchangeUnion/xud/pull/1082/files#diff-f4e7c7d131597f6bc8620b941b31f01c, I think we should hours and block in the logs:
```
debug: route time lock: 80 BTC blocks (~13h)
debug: maker client lock buffer: 576 LTC blocks (~24h)
```
So above is missing the word `blocks` and the hours `(~24h)`

Reply:
I added the delay figures in terms of hours like in your examples above @kilrau.